content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/ups_credentials.rst

```diff
@@ -1,68 +1,210 @@
-=====================================================
-How to get UPS credentials for integration with Odoo?
-=====================================================
+===============
+UPS integration
+===============
 
-In order to use the Odoo UPS API, you will need:
+UPS is a shipping carrier service that integrates with Odoo to coordinate shipping to all regions.
+Once integrated, users can create shipping methods that estimates shipping costs and :doc:`generate
+labels <../advanced_operations_shipping/labels>`.
 
-- A UPS.com user ID and password
+.. seealso::
+   :doc:`third_party_shipper`
 
-- A UPS account number
+To set up the UPS shipping connector in Odoo, complete these steps:
 
-- An Access Key
+#. Create a UPS account and get the credentials
+#. Configure company information in UPS
+#. Set up shipping method in Odoo
 
 An Access Key is a 16 character alphanumeric code that allows access to
```

Comment:
This is the only time an access key is mentioned in this article...do we actually need it?
If so can we add more context around it, as it seems out of place after the overview list right before we get into the config.